### ISSUE TITLE
peer dependency on `inquirer` missing

### ISSUE THREAD

**Issue by gjmveloso** [aws-amplify/amplify-cli#363]
**Describe the bug**
 `@aws-amplify/cli@0.1.29` installation is warning about a peer dependency that is missing:
 > npm WARN inquirer-autocomplete-prompt@1.0.1 requires a peer of inquirer@^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.

Which makes `npm list -g to throw an error message:
> npm ERR! peer dep missing: inquirer@^5.0.0 || ^6.0.0, required by inquirer-autocomplete-prompt@1.0.1

**To Reproduce**
1. Install @aws-amplify/cli by `npm install -g @aws-amplify/cli
2. See the warning message after the installation

**Expected behavior**
`aws-amplify/cli` should be able to install itself without generating any warning and/or error messages.

**Desktop (please complete the following information):**
 - OS: macOS 10.13.6
 - Versions: 
    - `node@v11.0.0`
    - `npm@6.4.1`
    - `aws-amplify/cli@0.1.29`

**Comment by NYCSwan** [comment]
Any luck with this issue? I've run into the same problem. It won't display the 'type of app' or let me configure amplify. 
OS: mac 10.13.6
node: 11.0.0
yarn: 1.10.1
amplify/cli: 0.1.27, 0.1.29
amplify-react-native: 2.0.8
amplify: 1.1.8

I got the same warning using 1.29. When I rolled back to 1.27 I did not get the error but I still could not get the app init'd. Tried deleting node_mods, clearing yarn cache, xcode cache
